src/include/miopen/datatype.hpp

```diff
@@ -159,9 +172,15 @@ inline KernelBuildParameters GetDataTypeKBP(miopenDataType_t type)
         {"MIOPEN_USE_BFP16", use_bfp16},
         {"MIOPEN_USE_INT32", use_int32},
         {"MIOPEN_USE_RNE_BFLOAT16", use_rne_bfloat16},
+        {"MIOPEN_FP8_IEEE_EXPONENT_BIAS", MIOPEN_FP8_IEEE_EXPONENT_BIAS},
+        {"MIOPEN_FP8_CLIPPING", MIOPEN_FP8_CLIPPING},
```

Comment:
@JehandadKhan I guess you are aware of that this change invalidates KDB for all solvers that use `SetupFloats()`, right? I would recommend producing these macros only if `MIOPEN_USE_FP8` is defined to narrow impact at least.

/cc @CAHEK7

Reply:
@cderb Please see previous comment.